### ISSUE TITLE
SMA SunSpec

### ISSUE THREAD

**Issue by mntnmn** [macb74/sunspec-monitor#1]
I've downloaded this project and I have it running on Debian 8, once I removed 'provided' from 'javax' in the pom file.
I have modified it to monitor a three string connected SMA inverter. Total AC power and current out, frequency, and DC power from the three strings. Small changes to 'ModbusReaderThread.java' and 'index.html'.
SMA SunSpec is broken on theSunny Boy 5.0-US that I have. All the data registers are shifted down one number, ie. 40188 ...AC current, appears at 40187. Not all registers are implemented. SMA fault.

This project is obviously in beginning stages but it has a lot of potential.
It could use a setup routine or configuration file for setting the registers that are recorded. Some way to change the scales of the charts would be good. Generic names for the signals, like 'channel_1' would make it more versatile.

Thank-you for posting it.

**Comment by macb74** [comment]
Hi, this was just a test to get in working with a solaredge inverter. Please feel free to update it with informations of other inverters.

**Comment by mntnmn** [comment]
Hi,
It is a good base to grow on. I am learning the code as you wrote it
and am going to make the config.properties file able to set input
channels, labels, maybe other things that could be set. I fixed some
typos that seemed to not matter. I want to figure out how to have the
package self execute from the jar. Now I run it with mvn
-Dexec.mainClass=de.martinbussmann.sunspecmonitor.SunspecMonitor
exec:java

As soon as I'm happy with things and can document it I will post what
I came up with.

Cheers!
